src/studio/src/designer/backend.Tests/_TestData/Model/CSharp/Kursdomene_HvemErHvem_M_2021-04-08_5742_34627_SERES.expected.csharp.txt

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Newtonsoft.Json;
+namespace Altinn.App.Models
+{
+  public class HvemErHvem_M{
```

Comment:
This expected file is missing the XmlRootAttribute. Might cause issues after merge with master.

Reply:
I'll merge in master and have a look. The compare is done on the compiled classes and I don't thing that takes attributes into account. Perhaps a weakness in itself and that the test should be extended to cover.

Reply:
Probably depends on how the classes are compared. I would think attributes are included, but they require reflection.

Reply:
It's an object graph comparison, so the decorating attributes is excluded. Added assert to verify that the XmlRootAttribute is included. Not perfect as we can add way more asserts, but it's a start.